src/NUnitTestAdapterInstall/source.extension.vsixmanifest

```diff
@@ -3,7 +3,7 @@
   <Identifier Id="NUnitTestAdapterInstall..7c53286e-ac4f-485f-915c-5ec5a4c47b0c">
     <Name>NUnit Test Adapter</Name>
     <Author>Charlie Poole</Author>
-    <Version>2.1.0.4</Version>
+    <Version>2.1.0.0</Version>
```

Comment:
It's a mistake because I thought I had accidentally typed in the 4 myself. Looking in GitHub, I see that the current version is 2.1.0.4, which begs the question "What was our last release?" AFAIK it was 2.0..0.0.

@OsirisTerje will need to clarify this, but I suspect he was following the odd / even strategy and was incrementing the forth digit rather than the third. We should not merge this without first deciding what version it should really be.

@OsirisTerje one thing that could help us here would be if we had release notes showing what we have actually released and what internal versions they used.